share/mk/bsd.compat.mk

```diff
@@ -210,6 +210,43 @@ LIB64CCFLAGS+=	-mllvm -cheri-subobject-bounds-clear-swperm=2
 .endif # CHERI_SUBOBJECT_BOUNDS
 .endif
 
+# -------------------------------------------------------------------
+# CHERI benchmarking world
+.if ${MK_LIB64CB} != "no"
+.if ${COMPAT_ARCH:Maarch64*}
+HAS_COMPAT+=	64CB
+LIB64CB_MACHINE=	arm64
+LIB64CB_MACHINE_ARCH=aarch64cb
+LIB64CBCPUFLAGS=	-target aarch64-unknown-freebsd13.0
+LIB64CBCPUFLAGS+=	-march=morello -mabi=purecap-benchmark
+LIB64CB_MACHINE_ABI=	${MACHINE_ABI:Nptr*:Npurecap} purecap ptr128c benchmark
```

Comment:
Yes, because for a hybrid world MACHINE_ABI won't contain purecap, so you need to add it, but for a purecap world it will, and I didn't think it was a good idea to duplicate entries